commands.go

```diff
@@ -343,6 +343,11 @@ func cmdCreate(c *cli.Context) {
 		log.Fatal("You must specify a machine name")
 	}
 
+	name, err := ValidateHostName(name)
```

Comment:
Seems to me it'd be good to merge this with the block immediately above (ln 341), since an empty name would also fail the ValidateHostName check. Also it'd be useful to `cli.ShowCommandHelp(c, "create")` before erroring out.